test/common/test_runner.go

```diff
@@ -107,6 +108,11 @@ func Setup(t *testing.T, runInParallel bool) *Client {
 	return client
 }
 
+func makeK8sNamespace(baseFuncName string) string {
+	base := helpers.MakeK8sNamePrefix(baseFuncName)
+	return names.SimpleNameGenerator.GenerateName(base + "-")
```

Comment:
👍 This is much nicer than what I did in https://github.com/knative/eventing/pull/1884.